skyportal/models.py

```diff
@@ -170,3 +221,30 @@ def from_ascii(cls, filename, source_id, instrument_id, observed_at):
         return cls(wavelengths=data[:, 0], fluxes=data[:, 1],
                    source_id=source_id, instrument_id=instrument_id,
                    observed_at=observed_at)
+
+
+#def format_public_url(context):
```

Comment:
What should happen to this?

Reply:
Just leave it there for now, there's nowhere better for it at the moment.